common.gypi

```diff
@@ -40,6 +40,10 @@
           ['OS != "win"', {
             'defines': [ 'EV_VERIFY=2' ],
           }],
+          ['OS == "android"', {
+            'cflags': [ '-g', '-O0', '-fwrapv', '-fPIE' ],
```

Comment:
We only need PIE for android. I should have avoided copying these other flags from above.